ui/panels-plugin/src/plugins/gauge-chart/GaugeChartPanel.tsx

```diff
@@ -110,11 +110,13 @@ export function GaugeChartPanel(props: GaugeChartPanelProps) {
     chartWidth = GAUGE_MIN_WIDTH;
   }
 
+  const hasMultipleCharts = gaugeData.length > 1;
+
   return (
     <Stack
       direction="row"
-      spacing={2}
-      justifyContent="left"
+      spacing={hasMultipleCharts ? 2 : 0}
+      justifyContent={hasMultipleCharts ? 'left' : 'center'}
```

Comment:
I noticed while I was in there that gauge charts look a little off when they're left aligned when there's a single gauge chart. Adjusting the behavior to center it when there's just one. If folks like it better the other way, let me know, and I can undo this bit.

Reply:
SGTM, thanks for fixing this!